memory_743_750_753.x

```diff
@@ -0,0 +1,114 @@
+MEMORY
+{
+  /* This file is intended for parts in the STM32H743/743v/753/753v families (RM0433), */
```

Comment:
You can remove the references to the Rev V parts here, the memory layout was the same anyhow